src/test/pselect.c

```diff
@@ -0,0 +1,8 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "rrutil.h"
+
+int main(void) {
+  pselect(0, NULL, NULL, NULL, NULL, NULL);
```

Comment:
It looks like this pselect call should indeed block indefinitely, as the test timeout on the buildbot suggests...